src/Event/Http/HttpRequestEvent.php

```diff
@@ -20,15 +20,23 @@ final class HttpRequestEvent implements LambdaEvent
     private $headers;
     /** @var string */
     private $queryString;
+    /** @var float */
+    private $payloadVersion;
 
     public function __construct(array $event)
     {
-        if (! is_array($event) || ! isset($event['httpMethod'])) {
+        // version 1.0 of the HTTP payload
+        if (isset($event['httpMethod'])) {
+            $this->method = strtoupper($event['httpMethod']);
+        } elseif (isset($event['requestContext']['http']) && isset($event['requestContext']['http']['method'])) {
+            // version 2.0 - https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html#http-api-develop-integrations-lambda.proxy-format
```

Comment:
That URL is super useful, that's the first time I see it.

Could you add it to the comment of the class as well? That way it's very visible.

Reply:
Nps, done :)